FreeOTP/EditViewController.swift

```diff
@@ -158,7 +158,8 @@ class EditViewController : UIViewController, UITextFieldDelegate, UIImagePickerC
             })
         })
 
-        textField(issuer, shouldChangeCharactersIn: NSRange(), replacementString: "")
+        // Why does this cause a runtime error, and what is this even for?
+//        textField(issuer, shouldChangeCharactersIn: NSRange(), replacementString: "")
```

Comment:
I don't remember why it is there. Just remove it and put an extended comment in the git commit.

Reply:
This was a weird field validation idiom -- it was crashing due to a change in the language semantics, but I found the real problem and reverted this change.